system_setup/client/client.py

```diff
@@ -38,16 +39,10 @@ class SystemSetupClient(SubiquityClient):
         ]
 
     def __init__(self, opts):
-        # TODO WSL:
-        # 1. remove reconfigure flag
-        # 2. decide on which UI to show up based on existing user UID >=1000
-        #    (or default user set in wsl.conf?)
-        # 3. provide an API for this for the flutter UI to know about it
-        # 4. Add Configuration Base page before Advanced
-        # 5. Add language page
-        # self.variant = "wsl_configuration"
-        if opts.reconfigure:
+        if is_reconfigure(opts.dry_run):
+            self.variant = "wsl_configuration"
             self.controllers = [
+                "Welcome",
```

Comment:
We shouldn’t have the Welcome controller loaded in reconfigure mode (see the spec)

Reply:
Removed.